docs/installation/basic.md

```diff
@@ -49,22 +49,33 @@ sudo -u postgres psql
 
 * When inside psql, create a user for open-event and then using the user create the database.
 
+For ease development, you should create Postgres user with the same username as your OS account. If your OS login account is _john_, for example, you should create _john_ user in Postgres. By this, you can skip entering password when using database.
```

Comment:
This suggestion does not seem necessary to me, different users are created for a database to manage access to multiple databases. Although, users are free to use their choice of user, we cannot suggest them this :)